awx/main/tests/functional/api/test_job_template.py

```diff
@@ -118,6 +118,21 @@ def test_extra_credential_unique_type_xfail(get, post, organization_factory, job
     assert response.data.get('count') == 1
 
 
+@pytest.mark.django_db
+def test_create_with_forks_exceeding_maximum_xfail(alice, post, project, inventory, settings):
+    project.use_role.members.add(alice)
+    inventory.use_role.members.add(alice)
+    settings.MAX_FORKS = 10
+    response = post(reverse('api:job_template_list'), {
+        'name': 'Some name',
+        'project': project.id,
+        'inventory': inventory.id,
+        'playbook': 'helloworld.yml',
+        'forks': 11,
+    }, alice)
+    assert response.status_code == 400
```

Comment:
job templates have a high chance of hitting 400 errors, so I want to make sure this hits the one we want

```python
assert 'Maximum number of forks (10) exceeded' in str(response.data)
```

or something like that

Reply:
@AlanCoding good idea. It is done.